core/server/middleware/private-blogging.js

```diff
@@ -49,7 +49,9 @@ privateBlogging = {
         }
 
         // take care of rss and sitemap 404s
-        if (req.url.lastIndexOf('/rss', 0) === 0 || req.url.lastIndexOf('/sitemap', 0) === 0) {
+        if (req.path.lastIndexOf('/rss/', 0) === 0 ||
```

Comment:
**Note:** This means we still get a redirect from `/rss` to `/rss/` even in private blogging. Not sure if that's an issue. Would otherwise require an exact match on `/rss` in `req.path`.